recipes/wagtail/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "wagtail" %}
+{% set version = "1.11.1" %}
+{% set sha256 = "fb54795d91c79cbaacb40ccfdbf047fa31a49fb65f57ec205d3656af4a9372b7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+    - django-modelcluster >=3.1,<4.0
+    - django-taggit >=0.20,<1.0
+    - django-treebeard >=3.0,<5.0
+    - djangorestframework >=3.1.3
+    - beautifulsoup4 >=4.5.1
+    - html5lib >=0.999,<1
+    - unidecode >=0.04.14
+    - willow >=0.4,<0.5
+    - requests >=2.11.1,<3.0
+   
+test:
```

Comment:
Please test the entry point:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands